spec/factories/users.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :user do
+    name { Faker::Name.name }
+    street_address { Faker::Address.street_address }
+    city { Faker::Address.city }
+    state {Faker::Address.state }
+    zip_code {Faker::Address.zip }
+    email { Faker::Internet.email }
+    password { Faker::Internet.password }
+    role { 0 }
```

Comment:
We'll need to add some users with role 0, some with role 1 (merchants), and just a couple with role 2(admin)